### ISSUE TITLE
** 3.10 testing **

### ISSUE THREAD

**Issue by bdraco** [fafhrd91/aiohttp#8529]
We've introduced a few significant changes in 3.10, so it would be great to get some community testing before promoting it as a stable release. We've released 3.10.0rc0 as a prerelease to help with this.

Can be installed with: `pip install 'aiohttp<4' --pre --upgrade`

In particular, these are things we are interested on testing or gathering feedback on:

- Implement happy eyeballs https://github.com/aio-libs/aiohttp/pull/8005
- Improved URL handler resolution time by indexing resources in the UrlDispatcher. For applications with a large number of handlers, this should increase performance significantly. https://github.com/aio-libs/aiohttp/pull/7917
- Restore AsyncResolver to be the default resolver. https://github.com/aio-libs/aiohttp/pull/8522 https://github.com/aio-libs/aiohttp/pull/8270
- Changes to the shutdown logic Shutdown logic: Only wait on handlers (https://github.com/aio-libs/aiohttp/pull/8530 .. changelog details forthcoming)

Release checklist
- [x] docs/changelog/tests for https://github.com/aio-libs/aiohttp/pull/8495
- [x] https://github.com/aio-libs/aiojobs/pull/495
- [x] aiojobs https://github.com/aio-libs/aiojobs/releases/tag/v1.3.0

Releases
- https://github.com/aio-libs/aiohttp/pull/8531 https://github.com/aio-libs/aiohttp/actions/runs/10044635669
- https://github.com/aio-libs/aiohttp/pull/8533 https://github.com/aio-libs/aiohttp/actions/runs/10045374258
- https://github.com/aio-libs/aiohttp/pull/8545 https://github.com/aio-libs/aiohttp/actions/runs/10145560535
- https://github.com/aio-libs/aiohttp/pull/8550 https://github.com/aio-libs/aiohttp/actions/runs/10163532577 (cancelled)
- https://github.com/aio-libs/aiohttp/pull/8552 https://github.com/aio-libs/aiohttp/actions/runs/10168515264

**Comment by bdraco** [comment]
Bumped both my Home Assistant large production and all good https://github.com/home-assistant/core/pull/122409

**Comment by bdraco** [comment]
As the testing period has ended, and we are moving to release, please report any new issues by creating a new issue report https://github.com/aio-libs/aiohttp/issues/new/choose

**Comment by webknjaz** [comment]
> Can be installed with: `pip install 'aiohttp<4' --pre --upgrade`

FYI, that `< 4` is not really needed anymore as I yanked all v4 pre-releases since many people complained.

**Comment by webknjaz** [comment]
I added a few comments on the change log in the related PR, in case you'd want to address them before proceeding with the final release.

**Comment by bdraco** [comment]
I canceled https://github.com/aio-libs/aiohttp/actions/runs/10163532577 and will do the cleanup when I get back home later tonight and re-do the release process

**Comment by webknjaz** [comment]
Thanks :pray:
